src/104/p104.py

```diff
@@ -0,0 +1,14 @@
+import math
+
+pand2 = lambda x: '0' not in x and len(set(x)) == 9
+pand1 = lambda x: x % 3 == 0 and pand2(str(x))
```

Comment:
Cannot simply pass as a set to `pand2`? Like `pand2(set(x))` and there we do the same thing, but without a string comparison.

What do you think? (: